### PR TITLE
DOCSP-22901 units format

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://mongodbcom-cdn.website.staging.corp.mongodb.com/docs-qa/docs-meta/docsworker-xlarge/DOCSP-22901-units-format/style-guide/style/numbers.html)

[JIRA](https://jira.mongodb.org/browse/DOCSP-22901)

### CODE REVIEW

File: source/style-guide/style/numbers.txt [hunk 1]
```diff
@@ -83,13 +86,14 @@ parenthetically after the numeral.
...
-       Enter **0** (zero). *(acceptable)*
      - two (2) panels
 
-       zero (0) probability
+   * - zero probability
+     - | zero (0) probability
+       | Enter **0** (zero). *(acceptable)*
+
+   * - Numbers with units
+     - 10MB
```
Comment (jeff-allen-mongo) [inline source/style-guide/style/numbers.txt:96]: My understanding of this table is that the "Use" and "Don't Use" columns are meant to show two variations on the same example. The "Use" column should be the accepted version of the copy, not a description.

Also, the lead-in to this table talks about numerals in parentheses, which wouldn't apply to this example.

I don't think this really belongs in this table.

Comment (davemungo) [reply source/style-guide/style/numbers.txt]: Done

**Review by davemungo — COMMENTED** [review]
Thanks @jeff-allen-mongo  this is ready for another pass

**Review by jeff-allen-mongo — APPROVED** [review]
LGTM, thanks for incorporating this update.

**Pull request merged**